blocks/library/gallery/index.js

```diff
@@ -61,16 +70,11 @@ registerBlockType( 'core/gallery', {
 			query( 'div.blocks-gallery figure.blocks-gallery-image img', {
 				url: attr( 'src' ),
 				alt: attr( 'alt' ),
-			} ),
+			} ) || [],
```

Comment:
>@aduth if others’ code isn't part of a webpack build how would they import `registerBlockType `, so that they have access to the API?

Each entry point (top-level directory) exposes itself on the `wp` global object:

https://github.com/WordPress/gutenberg/blob/740b93d/webpack.config.js#L27-L31

So by depending on `wp-blocks`, a plugin author can merely register their block using `wp.blocks.registerBlock` from the global scope.

Also documented at: https://github.com/WordPress/gutenberg/tree/master/blocks